pom.xml

```diff
@@ -247,6 +247,15 @@ under the License.
       <version>${resolverVersion}</version>
       <scope>provided</scope>
     </dependency>
+    <!-- incremental build support (http://www.eclipse.org/m2e/documentation/m2e-making-maven-plugins-compat.html) -->
+    <dependency>
+      <groupId>org.sonatype.plexus</groupId>
+      <artifactId>plexus-build-api</artifactId>
+      <version>0.0.7</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
unmaintained dependency ...

Reply:
Not really, it saw a new release at https://github.com/codehaus-plexus/plexus-build-api. The new GAV and packages are just not compatible with m2e yet (https://github.com/eclipse-m2e/m2e-core/issues/944).

Reply:
Compare also with the discussion at https://lists.apache.org/thread/h2gy5l7j5bv7ycbpwdjby66v8gwhzck2.